source/Octopus.Server.Client/Model/Observability/RegisterKubernetesMonitorCommand.cs

```diff
@@ -0,0 +1,18 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace Octopus.Client.Model.Observability;
+
+public class RegisterKubernetesMonitorCommand
+{
+    /// <summary>
+    /// Installation ID of that uniquely identifies the physical installation of the agent.
+    /// </summary>
+    [Required]
+    public string InstallationId { get; set; }
+
+    /// <summary>
+    /// Machine ID of that uniquely identifies the deployment target or worker that is being observed
```

Comment:
We should probably specify that this must be a Kubernetes Agent - it can't be a worker.